### PR TITLE
Layer layout navigation node pagination

### PR DESCRIPTION
**Pull request by munkk**

### CODE REVIEW

File: src/core/body/body.view.d.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+export declare interface IGetValueFactory{
+	(column: ColumnModel): IGetResult;
+}
+
+export declare class BodyView extends View {
+	constructor(public model: Model, public table: Table);
+
+	table: Table;
+	rows: any[];
```
Comment (vkorolev) [inline src/core/body/body.view.d.ts:16]: not any, maybe RowModel?

File: src/core/column-list/column.list.model.d.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import {ColumnModel} from "../column-type/column.model";
+
+export declare enum Generation{
```
Comment (vkorolev) [inline src/core/column-list/column.list.model.d.ts:3]: Better make it a type like this:
`type Generation = 'deep' | 'shallow';`
https://basarat.gitbooks.io/typescript/docs/types/literal-types.html

**Comment by vkorolev** [comment]
I think I saw some these changes in previous PR's. Can you please merge these branches or change PR destination to simplify review? 
Tell us if need any help with this :)

File: src/core/column-list/column.list.model.d.ts (hunk 2 above)
Comment (munkk) [reply src/core/column-list/column.list.model.d.ts:3]: thx)

**Pull request closed without merging**